src/databricks/labs/ucx/assessment/crawlers.py

```diff
@@ -139,6 +221,26 @@ def _assess_jobs(self, all_jobs: list[BaseJob], all_clusters_by_id) -> list[JobI
                 for value in cluster_config.spark_conf.values():
                     if "dbfs:/mnt" in value or "/dbfs/mnt" in value:
                         job_assessment[job.job_id].add(f"using DBFS mount in configuration: {value}")
+
+                # Checking if Azure cluster config is present in spark config
+                for key in cluster_config.spark_conf.items():
+                    if azure_sp_conf_usage_check(str(key)):
+                        job_assessment[job.job_id].add(f"{_AZURE_SP_CONF_FAILURE_MSG} Job cluster.")
+                        break
+
+            # Checking if Azure cluster config is present in cluster policies
+            if cluster_config.policy_id:
+                job_cluster_policy_definition = self._ws.cluster_policies.get(cluster_config.policy_id).definition
```

Comment:
same comment about policy variable here.

Reply:
I am only checking the policy family overrides. policy families are pre defined and will not have azure config.